LICENSE

```diff
@@ -1,4 +1,19 @@
-Copyright 2020 Disney Streaming Services
+Copyright 2024 Typelevel and respective contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+--------------------------------------------------------------------------
+Copyright 2020-2024 Disney Streaming Services
```

Comment:
I'd move this up to top of the file or otherwise I feel it'll never be updated. Maybe this:

```
Copyright 2024 Typelevel and respective contributors
Copyright 2020-2024 Disney Streaming Services
```

Then when we update to 2025, both get updated.

Reply:
I agree with moving the copyright up to the top so they're in pairs. This also avoids having to repeat the license fragment. Additionally, I would leave the DSS copyright backdated, for exactly the reasons @Baccata stated. Given that it's a fork, Disney has no claim on contributions made beyond that date.

Regarding the Typelevel wording, I still have concerns about the "and respective contributors" bit (and all variants thereof). That's not the only reason the heading is problematic though, so these are all shades of legal gray. I understand "Typelevel" (without qualification) to refer to the broad collective of individuals who have touched the specific file, since Typelevel is no longer a legally incorporated entity nor has it established a work-for-hire arrangement with any of the constituent contributors. All of which is to say that the copyright is *actually* held by 1) Disney (for anything pre-fork either performed by Disney employees or covered under CLA), and 2) the individual personages who wrote the code, which makes the whole header kind of wrong *unless* we interpret "Typelevel" to be a way of referring to the amorphous collective loosely identified by `git blame` rather than a singular indemnifying entity.

Reply:
Thanks for the feedback. I've moved the copyright up to the top, replaced `Typelevel` with just `respective contributors`, and deleted the additional Apache License block.

Note that by deleting the additional License block, we're keeping to Disney Streaming's trademark edits of Section 6. Is that really something we want? I separated the two initially (and moved Disney streaming down the page) so that we could have the standard license.

Reply:
Yeah once again, this is something that we could use @djspiewak's opinion on. 

Daniel, the license of disneystreaming/weaver is essentially a copy of Apache 2, with an additional amendment giving some strict wording with regards to branding. This amended Apache 2 was more recently named `TOST` ( https://disneystreaming.github.io/TOST-1.0.txt).

I think generally speaking, you cannot distribute software under a less strict license than the one it was originally distributed under. That being said, the paragraph feels way overzealous.